pastastore/base.py

```diff
@@ -1095,28 +1095,28 @@ def _set_series_name(series, name):
 
     @staticmethod
     def _check_model_series_names_for_store(ml):
+        prec_evap_model = ["RechargeModel", "TarsoModel"]
         if isinstance(ml, ps.Model):
             series_names = [istress.series.name
                             for sm in ml.stressmodels.values()
-                            if sm._name != "RechargeModel"
+                            if sm._name not in prec_evap_model
                             for istress in sm.stress]
-            if "RechargeModel" in [i._name for i in ml.stressmodels.values()]:
+            if prec_evap_model in [i._name for i in ml.stressmodels.values()]:
```

Comment:
You're probably working on it, but you can do one of these instead of what its showing now:

You can try set logic
```python
len(set(prec_evap_model) & set([i._name for i in ml.stressmodels.values()])) > 0:
```
or add an if statement to the list comprehension
```python
 if len([i._name for i in ml.stressmodels.values() if i._name in prec_evap_model]) > 0:
```

Reply:
Yes previously it was checked if the stressmodel name was `RechargeModel`. However it also needs to check if it is `TarsoModel` (or `ReservoirModel` in the future). So now I check if the stressmodel name is in the list `["RechargeModel", "Tarsomodel"]`. That was for now the quickest solution but maybe your suggestion is better.

Reply:
The in statement isnt doing what you think though, hence my suggestion :)
```python
["A" "b"] in ["A", "c", "b"]  # returns False
```

Reply:
Ah yes, I did not think thought about that :)). I was already happy that it worked. I could also use `numpy.isin()`?